source/core/src/main/com/csse3200/game/screens/MiniGameMenuScreen.java

```diff
@@ -0,0 +1,193 @@
+
+
+        TextButton snakeButton = new TextButton("Snake", buttonStyle);
+        TextButton skyButton = new TextButton("Flappy bird", buttonStyle);
+        TextButton waterButton = new TextButton("Underwater maze", buttonStyle);
+
+
+        Table table = new Table();
+        table.setFillParent(true);
+        table.center();
+
+
+        table.add(snakeImage).pad(10).row();
+        table.add(snakeButton).pad(10).row();
+        table.add(skyImage).pad(10).row();
+        table.add(skyButton).pad(10).row();
+        table.add(waterImage).pad(10).row();
+        table.add(waterButton).pad(10).row();
+
+
+        stage.addActor(table);
+
+
+        snakeImage.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                // Change button color to green when clicked
+                snakeImage.setColor(Color.GREEN);
+                // Switch to SnakeScreen
+                game.setScreen(new SnakeScreen(game));
+            }
+        });
+
+        skyImage.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                // Change button color to green when clicked
+                skyImage.setColor(Color.GREEN);
+                // Switch to SkyScreen
+              //  game.setScreen(new SkyScreen(game));
+            }
+        });
+
+        waterImage.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                // Change button color to green when clicked
+                waterImage.setColor(Color.GREEN);
+                // Switch to WaterScreen
+              //  game.setScreen(new WaterScreen(game));
+            }
+        });
+
+
+        snakeButton.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                // Change button color to green when clicked
+                snakeButton.setColor(Color.GREEN);
+                // Switch to SnakeScreen
+                game.setScreen(new SnakeScreen(game));
+            }
+        });
+
+        skyButton.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                // Change button color to green when clicked
+                skyButton.setColor(Color.GREEN);
+                // Switch to SkyScreen
+              //  game.setScreen(new SkyScreen(game));
+            }
+        });
+
+        waterButton.addListener(new ClickListener() {
+            @Override
+            public void clicked(InputEvent event, float x, float y) {
+                // Change button color to green when clicked
+                waterButton.setColor(Color.GREEN);
+                // Switch to WaterScreen
+              //  game.setScreen(new WaterScreen(game));
```

Comment:
remove commented code

Reply:
It is kept there for future implementation of other minigames.

Reply:
See above